scripts/uberenv/packages/axom/package.py

```diff
@@ -53,6 +53,12 @@ class Axom(CMakePackage, CudaPackage):
 
     version('main', branch='main', submodules=True)
     version('develop', branch='develop', submodules=True)
+
+    # SERAC EDIT START
+    version('0.4.0p1', SHA='fd2edc420f1ab3504c4890654266039b8966bb37', submodules="True")
+    # SERAC EDIT END
```

Comment:
He did it!

Reply:
kicking and screaming i will be brought into the future...